app/controllers/v1/pension_ipf_callbacks_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'decision_review_v1/utilities/logging_utils'
+
+module V1
+  class PensionIpfCallbacksController < ApplicationController
+    include ActionController::HttpAuthentication::Token::ControllerMethods
+    include DecisionReviewV1::Appeals::LoggingUtils
+
+    service_tag 'pension-ipf-callbacks'
+
+    skip_before_action :verify_authenticity_token, only: [:create]
+    skip_before_action :authenticate, only: [:create]
+    skip_after_action :set_csrf_header, only: [:create]
```

Comment:
Why are we disabling the CSRF protection here?

Reply:
There is no frontend associated with this endpoint and will not be accessible by any users. This endpoint will be triggered by VANotify and will use a bearer token for authentication.

Reply:
AH thanks for the context!